package.json

```
@@ -1,30 +1,27 @@
 {
   "name": "viktorbot",
-  "version": "2.6.0",
-  "description": "Custom bot for the /r/viktormains Discord channel!",
+  "version": "2.0.0",
```

PR mentions a 3.0.0 version though
```suggestion
  "version": "3.0.0",
```

---

Fixed in b872cb1